packages/bierzo-wallet/src/routes/account/transfer/components/NameTransferBack.tsx

```diff
@@ -0,0 +1,105 @@
+import { Fee, Identity, TransactionId } from "@iov/bcp";
+import { bnsCodec } from "@iov/bns";
+import { JsonRpcRequest } from "@iov/jsonrpc";
+import { FormValues, List, ListItem, makeStyles, Typography } from "medulas-react-components";
+import * as React from "react";
+
+import { history } from "../../..";
+import {
+  generateTransferAccountTxRequest,
+  generateTransferAccountTxWithFee,
+} from "../../../../communication/requestgenerators";
+import { RpcEndpoint } from "../../../../communication/rpcEndpoint";
+import { BwAccountWithChainName } from "../../../../components/AccountManage";
+import AccountOperation from "../../../../components/AccountOperation";
+import { STARNAME_MANAGE_ROUTE } from "../../../paths";
+
+const NAME_TRANSFER_BACK_ID = "name-transfer-back-id";
+
+interface HeaderProps {
+  readonly account: BwAccountWithChainName;
+}
+
+const Header: React.FunctionComponent<HeaderProps> = ({ account }): JSX.Element => (
+  <React.Fragment>
+    <Typography color="default" variant="h5" inline>
+      Transfer{" "}
+    </Typography>
+    <Typography color="primary" variant="h5" inline>
+      {`${account.name}*${account.domain}`}
```

Comment:
Sorry for misleading you. This is very odd construction, I will fix it in my code too.
I think much better will be: `{account.name}*{account.domain}`

Reply:
Oh sure, just removed the unneeded parts without thinking much 😅